ramls/samples/profiles/jobProfile.sample

```diff
@@ -0,0 +1,29 @@
+{
+   "id":"448ae575-daec-49c1-8041-d64c8ed8e5b1",
```

Comment:
```suggestion
   "id": "448ae575-daec-49c1-8041-d64c8ed8e5b1",
```
Not a big deal but could you please format JSON with space after the colon for consistency. I see that at least in src/main/resources/templates/db_scripts/schema.json this way is used.

Reply:
Reformatted all objects